File: corelib/src/main/java/org/apache/mvp/MainView.java
package org.apache.mvp;

import org.apache.cordova.domen.UIVisibleDataset;
import org.apache.mvp.presenter.DeviceCheck;

/*
 * Rotate function
 *
 * */
public interface MainView extends CompatView, DeepLinker {
    void dappend(String s);

    UIVisibleDataset data(); //Initial data values

    void makeScreen(UIVisibleDataset screen);

    boolean rotated();

    void hiDeRefreshLayout();

    //void wrapContentRequest();

    DeviceCheck checkDevice();


}

File: kwk/corelib/src/main/java/org/apache/mvp/MainView.kt
package org.apache.mvp

import org.apache.cordova.domen.UIVisibleDataset
import org.apache.mvp.presenter.DeviceCheck

/*
* Rotate function
*
* */
interface MainView : CompatView, DeepLinker, ReferrerAdapter.Callback {
    fun data(): UIVisibleDataset? //Initial data values

    fun makeScreen(screen: UIVisibleDataset)

    fun rotated(): Boolean

    fun hiDeRefreshLayout()

    //void wrapContentRequest();
    fun checkDevice(): DeviceCheck?
}
